### PR TITLE
[13.0][MIG] stock_request_purchase

### PR DESCRIPTION
**Pull request by JoanSForgeFlow**
Standard **stock_request_purchase** migration

Depends on:
- #839 

cc ~ @HviorForgeFlow @LoisRForgeFlow

### CODE REVIEW

File: stock_request_purchase/tests/test_stock_request_purchase.py [hunk 1]
```diff
@@ -158,17 +158,19 @@ def test_create_request_02(self):
...
+            self.stock_request_manager
+        ).create(vals)
 
         stock_request_1.action_confirm()
         self.assertEqual(
             sum(stock_request_1.sudo().purchase_line_ids.mapped("product_qty")), 5
         )
 
-        stock_request_2.action_confirm()
+        stock_request_2.sudo().action_confirm()
```
Comment (HviorForgeFlow) [inline stock_request_purchase/tests/test_stock_request_purchase.py]:
```suggestion
        stock_request_2.with_user(self.stock_request_manager).action_confirm()
```

File: stock_request_purchase/views/purchase_order_views.xml [hunk 2]
```diff
@@ -19,7 +19,7 @@
                     <field name="stock_request_ids" invisible="1"/>
                 </button>
             </xpath>
-            <xpath expr="//field[@name='order_line']/form/sheet/notebook"
+            <xpath expr="//field[@name='order_line']/form//notebook"
```
Comment (HviorForgeFlow) [inline stock_request_purchase/views/purchase_order_views.xml]:
```suggestion
            <xpath expr="//field[@name='order_line']/form/notebook"
```

**Review by HviorForgeFlow — COMMENTED** [review]
Code review

**Review by LoisRForgeFlow — APPROVED** [review]
Tested locally, works good!

Can you update the icon of the module with the newest one (check stock_request module in this version)?

**Review by AaronHForgeFlow — APPROVED** [review]
Functional test on local :+1:

**Pull request merged**